### PR TITLE
check length of right-hand column before attempting to load table

### PR DESCRIPTION
**Pull request by NathanielBennett**
Fixes the problem currently seen here: http://www.theguardian.com/football/2015/jan/04/david-moyes-real-sociedad-barcelona-match-report where there is insufficient space for the league table to render in the righht hand column. Checks the available height of the right hand column and only displays the league table where there's enough space to do so - similarly to branded commercial components here:  https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/creatives/branded-component.js#L51

This causes the shorter, results table to be loaded.

Before: 
badly-rendered-table

After: 
fixed-table

### CODE REVIEW

**Comment by ironsidevsquincy** [comment]
Are you saying if the RHC is longer than 1800px, show the smaller table? Should it be something like 'If RHC is > 1000 && < 18000 show shorter table, otherwise if > 1800 show longer table'

File: static/src/javascripts/bootstraps/football.js [hunk 1]
```diff
@@ -226,7 +226,10 @@ define([
         });
 
         page.isCompetition(function (competition) {
-            renderTable(competition, extras, dropdownTemplate);
+            var $rightHandCol = $('.js-secondary-column')
+            if ($rightHandCol.dim().height > 1800 ) {
```
Comment (phamann) [inline static/src/javascripts/bootstraps/football.js]: Maybe leave a comment above this conditional to explain why we are having to do this.

**Comment by phamann** [comment]
After discussing with @jamesgorrie we feel that this is the correct logic

**Comment by phamann** [comment]
:+1:

**Comment by jamesgorrie** [comment]
Just to answer @ironsidevsquincy's question - he is saying if the column is bigger than 1800, show the big table. The small table is shown irrespective.

**Pull request merged**